### DISCUSSION TITLE
2023-09-26

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#574] [daily_meeting]
### 오늘의 할 일
- 프론트엔드
  - 오전 강의

- 백엔드
  - 오전 강의

- 공통
  - 오후에 회의(업무 분담 및 사용자 유치 계획 실행)
  - 회의 이후에는 각자 개발 진행

- 개인 일정
  - 노아(5시~6시 근로)

### 내일의 할 일

**Comment by woosung1223** [comment]
### 체크인
9점 - 오늘 기분이 낫배드 하네요 굿 사용자가 생겨서 좋아요

### 체크아웃
9점 - 나쁘지 않군요

**Comment by nlom0218** [comment]
### 체크인

10점 - 비가 왼 시원하고 좋네요

### 체크아웃

8점 - 6퇴를 할까 말까 고민이에요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 사용자 유치 시작하니까 재밌네요 ㅎㅎ

### 체크아웃
9점 - 폴링 얼추 구현 끝나니까 좋네요

**Comment by aak2075** [comment]
### 체크인

10점 - 스터디 첫 날부터 늦었네요ㅋㅋㅋ

### 체크아웃

10점 - 간만에 코딩 열심히 했네요

**Comment by woo-jk** [comment]
### 체크인
9점 - 시원해서 좋아요

### 체크아웃
7점 - 미션 하면서 너무 삽질했어요,,

**Comment by jaehee329** [comment]
### 체크인
6점 - 조금씩 수면이 줄어드는...
### 체크아웃
10점 - 내일만 하면 연휴? 미쳐따

**Comment by yeopto** [comment]
### 체크인

7점 - 장트러블 이슈

### 체크아웃

10점 - 미션 제출해서 마음이 한결 편해지네요.
